extensions/amazon-lambda/event-server/src/main/java/io/quarkus/amazon/lambda/runtime/MockEventServer.java

```diff
@@ -0,0 +1,209 @@
+package io.quarkus.amazon.lambda.runtime;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.LinkedBlockingQueue;
+
+import org.jboss.logging.Logger;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpServer;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.BodyHandler;
+
+public class MockEventServer implements Closeable {
+    protected static final Logger log = Logger.getLogger(MockEventServer.class);
+    public static final int PORT = Integer.getInteger("quarkus-internal.aws-lambda.test-port", 8081);
```

Comment:
As mentioned elsewhere IMHO this should be a proper config object, and should be different for dev and test.